src/trace-api.js

```diff
@@ -177,6 +178,22 @@ TraceAgent.prototype.runInRootSpan = function(options, fn) {
 };
 
 /**
+ * Returns a unique identifier for the current root span. This can be used to
+ * uniquely identify the current root span. If there is no current root span,
+ * or if we have lost context, this will return null. The structure and the
+ * length of the returned string should be treated opaquely - the only guarantee
+ * is that the value would unique for every root span.
+ * @returns {string} the root span id, or null if it doesn't exist
+ */
+TraceAgent.prototype.getRootSpanId = function() {
```

Comment:
The suggestion does make sense from a trace service API point of view, however, the [public JS API ](https://github.com/GoogleCloudPlatform/cloud-trace-nodejs/blob/master/doc/trace-api.md#trace-spans)we expose from the module uses concepts like `runInRootSpan`. We don't differentiate Root Trace and Root Span to our users, so `RootSpanId` might make more sense. WDYT?